oat-sa/extension-tao-testqti-previewer
# Fix/AUT-1879/disable token store for test preview

Related to: https://oat-sa.atlassian.net/browse/AUT-1879

With the test previewer, we need to disable the `tokenStore` when initializing the test runner.
This PR also makes sure the `tokenStore` is disabled when calling additional actions from the item previewer.

Additionally, a few code-style issues have been addressed at the same time.

**Note:**
The unit tests were failing for some time, and I had to fix them. As this could be a lot of code to read, here are the 2 main commits that contain the fix to look at:
-  [fix: disable the tokenStore when initializing the test preview](https://github.com/oat-sa/extension-tao-testqti-previewer/pull/161/commits/1d6803208d57cbe99653fea162b9debf40408adf)
- [fix: disable the tokenStore when calling item and test actions from the item preview](https://github.com/oat-sa/extension-tao-testqti-previewer/pull/161/commits/b737e22762d4b5ff1c42281e37f33e6c171a7658) 

**How to test**
- checkout the branch
- log in as a back-office user
- preview an item (it should work seamlessly)
- preview a test, then close it
- repeat the previous step more than 10 times. Without the fix, it fails on the 11th try.

Unit tests should pass:
```
cd tao/views/build
npm ci
npx grunt connect:dev taoqtitestpreviewertest
```

Impacted unit tests:
- `taoQtiTestPreviewer/views/js/test/previewer/proxy/item/test.html`
- `taoQtiTestPreviewer/views/js/test/previewer/proxy/test/test.html`

They can be tested from the browser as well
